src/Containers/MassTransit.AutofacIntegration/MassTransit.AutofacIntegration.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac" Version="5.1.2" />
+    <PackageReference Include="Autofac" Version="5.1.4" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-*" PrivateAssets="All" />
```

Comment:
Might as well update the SourceLink.Github to the full version 1.0.0 in all csproj while you are in here.

Reply:
Is that a thing? I don’t understand all these version references.

I do need to change the package references for MassTransit to be <= 6 at some point, particularly for those packages that are going away in v7.

Do you know how to do that? For Unity and Ninject, if you could get those so they only work with [6.2.6,7.0) that would be great.

Reply:
Because we are using dotnet pack (with the csproj nuget pack properties), as far as I can tell we cannot supply a version constraint.